addons/web_editor/views/snippets.xml

```diff
@@ -464,74 +464,146 @@
-                        <we-button t-if="not no_animations" data-set-img-shape="web_editor/basic/bsc_organic_2" data-select-label="Organic Medium" data-animated="true"/>
-                        <we-button t-if="not no_animations" data-set-img-shape="web_editor/basic/bsc_organic_3" data-select-label="Organic Hard" data-animated="true"/>
+                    <we-select-page string="Geometrics">
+                        <we-button data-set-img-shape="web_editor/geometric/geo_square" data-select-label="Square" />
+                        <we-button data-set-img-shape="web_editor/geometric/geo_shuriken" data-select-label="Shuriken" />
+                        <we-button data-set-img-shape="web_editor/geometric/geo_diamond" data-select-label="Diamond" />
+                        <we-button data-set-img-shape="web_editor/geometric/geo_triangle" data-select-label="Triangle" />
+                        <we-button data-set-img-shape="web_editor/geometric/geo_cornered_triangle" data-select-label="Corner Triangle" />
+                        <we-button data-set-img-shape="web_editor/geometric/geo_pentagon" data-select-label="Pentagon" />
+                        <we-button data-set-img-shape="web_editor/geometric/geo_hexagon" data-select-label="Hexagon" />
+                        <we-button data-set-img-shape="web_editor/geometric/geo_heptagon" data-select-label="Heptagon" />
+                        <we-button data-set-img-shape="web_editor/geometric/geo_star" data-select-label="Star" />
+                        <we-button data-set-img-shape="web_editor/geometric/geo_star_8pin" data-select-label="Star 2" />
+                        <we-button data-set-img-shape="web_editor/geometric/geo_star_16pin" data-select-label="Star 3" />
+                        <we-button data-set-img-shape="web_editor/geometric/geo_slanted" data-select-label="Slanted" />
+                        <we-button data-set-img-shape="web_editor/geometric/geo_emerald" data-select-label="Emerald" />
+                        <we-button data-set-img-shape="web_editor/geometric/geo_tetris" data-select-label="Tetris" />
+                        <we-button data-set-img-shape="web_editor/geometric/geo_kayak" data-select-label="Kayak" />
+                        <we-button data-set-img-shape="web_editor/geometric/geo_tear" data-select-label="Tear" />
+                        <we-button data-set-img-shape="web_editor/geometric/geo_cornered" data-select-label="Cornered" />
+                        <we-button data-set-img-shape="web_editor/geometric/geo_sonar" data-select-label="Sonar" />
+                        <we-button data-set-img-shape="web_editor/geometric/geo_obb" data-select-label="Obb" />
+                        <we-button t-if="not no_animations" data-set-img-shape="web_editor/geometric/geo_square_2" data-select-label="Square 2" data-animated="true"/>
```

Comment:
Why do we need the `t-if` on each and every animated shape?
Can't you just remove it from those `<we-button/>` and group them in a `<t t-if`?
It looks really weird, especially since we always put the animated ones at the end.
Even if it was there before, now is a good opportunity to clean that imo.
Eg instead of
```xml
                        <we-button data-set-img-shape="web_editor/geometric_round/geo_round_pill" data-select-label="Round Pill" />
                        <we-button data-set-img-shape="web_editor/geometric_round/geo_round_cornered" data-select-label="Round Cornered" />
                        <we-button data-set-img-shape="web_editor/geometric_round/geo_round_cornered2" data-select-label="Round Cornered 2" />
                        <we-button data-set-img-shape="web_editor/geometric_round/geo_round_obb" data-select-label="Round Obb" />
                        <we-button data-set-img-shape="web_editor/geometric_round/geo_round_sonar" data-select-label="Round Sonar" />
                        <we-button data-set-img-shape="web_editor/geometric_round/geo_round_clover" data-select-label="Round Clover" />
                        <we-button data-set-img-shape="web_editor/geometric_round/geo_round_clover2" data-select-label="Round Clover 2" />
                        <we-button t-if="not no_animations" data-set-img-shape="web_editor/geometric_round/bsc_square_rounded_1" data-select-label="Square Rounded 1" data-animated="true"/>
                        <we-button t-if="not no_animations" data-set-img-shape="web_editor/geometric_round/bsc_square_rounded_2" data-select-label="Square Rounded 2" data-animated="true"/>
                        <we-button t-if="not no_animations" data-set-img-shape="web_editor/geometric_round/bsc_organic_1" data-select-label="Organic Soft" data-animated="true"/>
                        <we-button t-if="not no_animations" data-set-img-shape="web_editor/geometric_round/bsc_organic_2" data-select-label="Organic Medium" data-animated="true"/>
                        <we-button t-if="not no_animations" data-set-img-shape="web_editor/geometric_round/bsc_organic_3" data-select-label="Organic Hard" data-animated="true"/>
```
Do
```xml
                        <we-button data-set-img-shape="web_editor/geometric_round/geo_round_pill" data-select-label="Round Pill" />
                        <we-button data-set-img-shape="web_editor/geometric_round/geo_round_cornered" data-select-label="Round Cornered" />
                        <we-button data-set-img-shape="web_editor/geometric_round/geo_round_cornered2" data-select-label="Round Cornered 2" />
                        <we-button data-set-img-shape="web_editor/geometric_round/geo_round_obb" data-select-label="Round Obb" />
                        <we-button data-set-img-shape="web_editor/geometric_round/geo_round_sonar" data-select-label="Round Sonar" />
                        <we-button data-set-img-shape="web_editor/geometric_round/geo_round_clover" data-select-label="Round Clover" />
                        <we-button data-set-img-shape="web_editor/geometric_round/geo_round_clover2" data-select-label="Round Clover 2" />
                        <t t-if="not no_animations">
                            <we-button data-set-img-shape="web_editor/geometric_round/bsc_square_rounded_1" data-select-label="Square Rounded 1" data-animated="true"/>
                            <we-button data-set-img-shape="web_editor/geometric_round/bsc_square_rounded_2" data-select-label="Square Rounded 2" data-animated="true"/>
                            <we-button data-set-img-shape="web_editor/geometric_round/bsc_organic_1" data-select-label="Organic Soft" data-animated="true"/>
                            <we-button data-set-img-shape="web_editor/geometric_round/bsc_organic_2" data-select-label="Organic Medium" data-animated="true"/>
                            <we-button data-set-img-shape="web_editor/geometric_round/bsc_organic_3" data-select-label="Organic Hard" data-animated="true"/>
                        </t>
```